CHANGES.md

```diff
@@ -1,5 +1,7 @@
 # Change Log
 
+- Parse `skip_magic_trailing_comma` from pyproject.toml (#2504)
```

Comment:
Should clarify that this only affects the vim plugin

Reply:
Clarified.